cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/extensions/SpConfigExt.kt

```diff
@@ -1,7 +1,13 @@
 package com.sourcepoint.cmplibrary.util.extensions
 
+import com.sourcepoint.cmplibrary.creation.ConfigOption
 import com.sourcepoint.cmplibrary.exception.CampaignType
 import com.sourcepoint.cmplibrary.model.exposed.SpConfig
 
 internal fun SpConfig.isIncluded(campaign: CampaignType) =
     campaigns.find { it.campaignType == campaign } != null
+
+internal fun SpConfig.hasTransitionCCPAAuth() =
+    campaigns.find { it.campaignType == CampaignType.USNAT }
+        ?.configParams
+        ?.find { it == ConfigOption.TRANSITION_CCPA_AUTH } != null
```

Comment:
Maybe we can use `contains` instead of `find`?

```kotlin
internal fun SpConfig.hasTransitionCCPAAuth() =
    campaigns.find { it.campaignType == CampaignType.USNAT }
        ?.configParams
        ?.contains(ConfigOption.TRANSITION_CCPA_AUTH) ?: false
```
(not sure if the code above compiles)

Reply:
In this case the usnat contains the TRANSITION_CCPA_AUTH, I need to find the campaign and the check if it contains that value

Reply:
Got it, thanks 👍